### PR TITLE
2023-update and mission-planning Merge

### PR DESCRIPTION
**Pull request by pizzalemon**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
 		"styled-components": "^5.1.1"
 	},
 	"scripts": {
-		"start": "react-scripts --openssl-legacy-provider start",
+		"start": "react-scripts start",
```
Comment (krishnans2006) [inline client/package.json]: This reverts a fix made in ba653ddf1d5b6a193f59bde26e8b1c9092573acb... did the fix break your version?

File: client/src/pages/FlightData/index.js [hunk 2]
```diff
@@ -84,7 +86,12 @@ const FlightData = () => {
 		obstacles: "Obstacles",
 		offAxis: "Off Axis ODLC",
 		searchGrid: "ODLC Search Grid",
+		loiter: "loiter",
```
Comment (krishnans2006) [inline client/src/pages/FlightData/index.js]: Capitalization?

File: server/apps/uav.py [hunk 3]
```diff
@@ -138,20 +138,6 @@ def uav_export_commands_file():
     return {"waypoints": waypoints}
 
 
-@uav_commands.route("/export")
```
Comment (krishnans2006) [inline server/apps/uav.py]: You sure this route is unnecessary?

Comment (pizzalemon) [reply server/apps/uav.py]: That route was a duplicate route (and also it was older and wrong). The correct export route should be right above it.

File: client/src/pages/FlightData/index.js (hunk 2 above)
Comment (pizzalemon) [reply client/src/pages/FlightData/index.js]: lmao

File: client/package.json (hunk 1 above)
Comment (pizzalemon) [reply client/package.json]: Undoing this now

**Review by krishnans2006 — APPROVED** [review]
lgtm

**Pull request merged**